src/app/shared/components/status-info-card/status-info-card.component.ts

```diff
@@ -1,19 +1,24 @@
 import { Component, Input } from '@angular/core';
 import { ApplicationIcons } from '../../enum/applications';
 import { ApplicationStatusDescription, ApplicationTitles } from '../../enum/enumUA/statuses';
-import { ApplicationStatuses } from '../../enum/statuses';
+import { ApplicationStatuses, ApplicationProviderStatuses } from '../../enum/statuses';
 import { Application } from '../../models/application.model';
+import { Role } from '../../enum/role';
 
 @Component({
   selector: 'app-status-info-card',
   templateUrl: './status-info-card.component.html',
   styleUrls: ['./status-info-card.component.scss']
 })
 export class StatusInfoCardComponent {
+  @Input() public role: Role;
```

Comment:
Make this input a setter so that we can keep one property for the status ( `public status: ApplicationStatuses | ApplicationProviderStatuses`) and set it based on the role and the template will remain as simple as before